ApiBundle/Controller/SiteController.php

```diff
@@ -62,11 +62,16 @@ public function listAction(Request $request)
         );
 
         $repository =  $this->get('open_orchestra_model.repository.site');
+        $transformer = $this->get('open_orchestra_api.transformer_manager')->get('site_collection');
+
+        if ($entityId = $request->get('entityId')) {
```

Comment:
Why does this transformation and search cannot be performed by the Trait directly?